src/features/app/interface.rs

```diff
@@ -1,16 +1,11 @@
 use crate::app::AppContext;
 
-use self::protobuf::{App, GetRatingRequest, GetRatingResponse};
-pub use protobuf::app_server;
+use crate::features::pb::app::{GetRatingRequest, GetRatingResponse};
 use tonic::{Request, Response, Status};
 
-use super::{service::AppService, use_cases};
-
-pub mod protobuf {
-    pub use self::app_server::App;
+use crate::features::pb::app::app_server::App;
 
-    tonic::include_proto!("ratings.features.app");
-}
+use super::{service::AppService, use_cases};
```

Comment:
Any reason to not `use crate::` here? I rarely see Rust crates `use super` except in unit tests, but I won't be too picky.